CONTRIBUTING.md

```diff
@@ -1 +1,138 @@
-# Contributing (TODO)
+# Contributing to semantic-release
+
+As a contributor, here are the guidelines we would like you to follow:
+
+- [Code of conduct](#code-of-conduct)
+- [How can I contribute?](#how-can-i-contribute?)
+- [Submitting a Pull Request](#submitting-a-pull-request)
+- [Coding rules](#coding-rules)
+- [Working with the code](#get-started)
+
+We also recommend that you read [How to Contribute to Open Source](https://opensource.guide/how-to-contribute).
+
+## Code of conduct
+
+Please read and follow our [Code of conduct](CODE_OF_CONDUCT.md).
+
+## How can I contribute?
+
+- Improve documentation
+- Give feedback on issues
+- Fix bugs and implement new features
+- [Bug report](https://github.com/teamsempo/SempoBlockchain/issues/new?assignees=&labels=&template=bug_report.md&title=)
+- [Feature request](https://github.com/teamsempo/SempoBlockchain/issues/new?assignees=&labels=&template=feature_request.md&title=)
+
+## Submitting a Pull Request
+
+TODO
+
+## Coding rules
+
+### Source code
+
+TODO
+
+### Documentation
+
+TODO
+
+### Commit message guidelines
+
+#### Atomic commits
```

Comment:
how much do we care about this given we squash merge? seems like this guideline is kinda spec bloat to me... we've never mentioned anything like this before

Reply:
I kind of read this as `atomic PRs`, given the squash. But I guess most PRs are atomic by nature anyway

Reply:
Yeah, I view it as atomic commits on master branch (so PRs)